### DISCUSSION TITLE
How to use ALLUSERS property in wix v5

### DISCUSSION THREAD

**Discussion by jmjaffe37** [wixtoolset/issues#8151] [Questions] [answered]
Hello,
I am attempting to update an opensource tool ([adoptium/insaller](https://github.com/adoptium/installer)) which is used to create installers for open source Java Development Kits. This tool is currently using wix 3.14, and I am updating it to wix v5.

Here's the problem: whenever I attempt to declare the `ALLUSERS` property, wix logs tell me that there is a duplicate symbol error ([link to sample run logs](https://github.com/adoptium/installer/actions/runs/8789003918/job/24117731433?pr=851#step:7:57)) and it fails. The duplicate symbol lines come from the `<Package Name="...` tag (which does not have `ALLUSERS` or installer scope specified), and the `<Property Id="ALLUSERS" Value="2" />` tag. 

I got around this in testing by renaming all instances of this variable to `ALLUSERSSETUP`, but I have recently learned that `ALLUSERS` is a specific property that has implications for the installation. (Here is a [link to the gh PR discussion](https://github.com/adoptium/installer/pull/851))

So, I wanted to ask if there is a way to initially set property `ALLUSERS` to 2 in a way that wix agrees with.
Thank you!

**Comment by chrpai** [comment]
This should help:

https://wixtoolset.org/docs/v3/xsd/wix/package/

**Reply by jmjaffe37** [reply, marked as answer]
Thank you! I am actually using wix v5, but I ended up finding these sets of resources which pointed me to setting the `Scope` property to `perUserOrMachine`

Package element:
https://wixtoolset.org/docs/schema/wxs/package/

PackageScopeType:
https://wixtoolset.org/docs/schema/wxs/packagescopetype/
